.github/workflows/build.yml

```diff
@@ -12,7 +11,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: false
```

Comment:
This sometimes fails on our schedule, so `${{ github.event_name == 'pull_request' }}`

Reply:
Saw a comment on the other PR that this field doesn't allow expressions, I pushed up a test PR and it looks like it does, but the JSON schema doesn't account for that. I'm also fine with keeping this as `false`. Most of the times failures are temporary issues and you can re-run just failed jobs

Reply:
Ok ty